tests/ci/run_benchmark_build_tests.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash -ex
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+source tests/ci/common_posix_setup.sh
+
+install_dir="$(pwd)/../librypto_install_dir"
```

Comment:
NP: to improve readability can this SRC_ROOT or BUILD_ROOT?

Reply:
This is where the OpenSSL build is being installed to